docs/docs/installation/installing-rasa-open-source.mdx

```diff
@@ -9,6 +9,25 @@ import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 import { Button } from "@theme/Button";
 
+
+
+The next generation of Rasa's dialogue management is currently only 
+available as a Rasa Labs release, which is restricted to customers of Rasa.
+
+If you already have a valid Rasa Pro license, you can install Rasa Labs
+releases using the installation instructions from the 
+[Rasa Pro Python Installation](../installation/rasa-pro/installation.mdx#python-package-installation) guide.
+
+After you've completed the authentication and configuration steps
+in the Rasa Pro Installation, you can install the latest Rasa Labs version
+using:
+
+```bash
+pip install rasa-plus==3.8.0a4
```

Comment:
that's not the latest alpha, right? maybe we can just use `pip install 'rasa-plus>3.7' --pre` and be done with it 😅